validation/process_rlimits.go

```diff
@@ -17,7 +17,12 @@ func main() {
 	if err != nil {
 		util.Fatal(err)
 	}
-	g.AddProcessRlimits("RLIMIT_NOFILE", 1024, 1024)
+	g.AddProcessRlimits("RLIMIT_AS", 100, 200)
```

Comment:
This is not many bytes ;).  Does it even run?  Similarly for `RLIMIT_DATA`, `RLIMIT_FSIZE`, and `RLIMIT_STACK` below.  I don't see a problem with making these generous values in the 1GB range, as long as they're distinct values.